src/core/Common.js

```diff
@@ -10,6 +10,7 @@ module.exports = Common;
 
 (function() {
 
+    Common._timeUnit = 1000 / 60;
```

Comment:
It would be great if this property dropped the `_`. There are some cases where I use it to make `Body.applyForce` framerate insensitive.

Example from my code:

```js
// deltaAdjustment is Common._timeUnit divided by the smallest delta over the last 30 frames
// (I'm using something similar as your Runner.tick here https://github.com/liabru/matter-js/blob/82bb41535b09e02dd76a20d5955fc7b4102487d6/src/core/Runner.js#L132)
const recoilFactor = -(body.mass * 0.09) * deltaAdjustment;
const recoil = Vector.mult(norm, recoilFactor);

// recoil
Body.applyForce(playerControlledBody, playerControlledBody.position, recoil);
```

Reply:
The reason is that it's an internal constant so your code shouldn't rely on it. `Body.applyForce` now accounts correctly so I guess you won't need it?